src/EditorFeatures/VisualBasicTest/InitializeParameter/AddParameterCheckTests.vb

```diff
@@ -58,6 +58,25 @@ class C
 end class")
         End Function
 
+        <WorkItem(47030, "https://github.com/dotnet/roslyn/issues/47030")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)>
+            Public Async Function TestOnByRefParameter() As Task
+            Await TestInRegularAndScript1Async(
+"
+class C
+    public sub new([||]byref s as string)
+    end sub
+end class",
+"
+class C
+    public sub new(byref s as string)
+        If s Is Nothing Then
+            Throw New System.ArgumentNullException(NameOf(s))
+        End If
+    end sub
+end class")
+        End Function
```

Comment:
can we have a VB test that at least documents the behavior with `<out>` params in vb?

Reply:
Yeah, we probably should have this; that OK @Youssef1313?